<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>817</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>511</width>
      <height>711</height>
     </rect>
    </property>
    <property name="title">
     <string>Individual movement</string>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>481</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>Table Control</string>
     </property>
     <widget class="QDoubleSpinBox" name="doubleSpinBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="minimum">
       <double>0.000100000000000</double>
      </property>
      <property name="maximum">
       <double>0.050000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.000100000000000</double>
      </property>
      <property name="value">
       <double>0.050000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Speed </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Direction</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>62</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>CW</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>120</y>
        <width>62</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>CCW</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>150</y>
        <width>221</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>Table Control</string>
      </property>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <property name="decimals">
        <number>5</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Speed </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Direction</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_13">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>90</y>
         <width>62</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>CW</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_14">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>120</y>
         <width>62</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>CCW</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>50</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Go</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>20</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Go Table </string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>70</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop Table </string>
      </property>
     </widget>
    </widget>
    <widget class="QDial" name="dial">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>101</width>
       <height>111</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>-1</number>
     </property>
     <property name="sliderPosition">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>271</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Actuator1</string>
     </property>
     <widget class="QRadioButton" name="radioButton_16">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_19">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>271</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Actuator2</string>
     </property>
     <widget class="QRadioButton" name="radioButton_17">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_18">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>271</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Servo2</string>
      </property>
      <widget class="QRadioButton" name="radioButton_22">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_23">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>271</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Actuator3</string>
     </property>
     <widget class="QRadioButton" name="radioButton_32">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_33">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_11">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>271</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Servo2</string>
      </property>
      <widget class="QRadioButton" name="radioButton_34">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_35">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>271</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Actuator4</string>
     </property>
     <widget class="QRadioButton" name="radioButton_36">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_37">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_13">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>271</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Servo2</string>
      </property>
      <widget class="QRadioButton" name="radioButton_38">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_39">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>460</y>
       <width>271</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Actuator5</string>
     </property>
     <widget class="QRadioButton" name="radioButton_40">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_41">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_15">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>271</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Servo2</string>
      </property>
      <widget class="QRadioButton" name="radioButton_42">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_43">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>530</y>
       <width>271</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Actuator6</string>
     </property>
     <widget class="QRadioButton" name="radioButton_48">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_49">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_18">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>271</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Servo2</string>
      </property>
      <widget class="QRadioButton" name="radioButton_50">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_51">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Smarties colors:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Requested colors:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Customer's name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
